doc/api/util.md

```diff
@@ -126,6 +126,10 @@ util.format(1, 2, 3); // '1 2 3'
 
 ## util.inherits(constructor, superConstructor)
 
+_Note: usage of `util.inherits()` is discouraged. Please use ES6 `class` and
+`extends` to get language level inheritance support, including an unbroken
+inheritance chain._
```

Comment:
That's some dark JS there. So in the last step you are overriding the prototype of a new object with a new object and eventually will return the former, whereas in the first case you attempt to mutate an existing objects prototype which does not yield proper inheritance? That's really intransparent (probably purposefully).

I don't know...shall I change it to ... then?:
_Note: usage of `util.inherits()` is discouraged. Please use the ES6 `class` and
`extends` keywords to get language level inheritance support. `util.inherits()` makes a call to `Object.setPrototypeOf()`, which by definition does not override prototypes entirely._